CMakeLists.txt

```diff
@@ -47,18 +47,18 @@ set(TRITON_CORE_REPO_TAG ${TRITON_BACKEND_API_VERSION} CACHE STRING "Tag for tri
 set(TRITON_BACKEND_REPO_TAG ${TRITON_BACKEND_API_VERSION} CACHE STRING "Tag for triton-inference-server/backend repo")
 
 set(DALI_VERSION "" CACHE STRING
-        "DALI version that should be downloaded by the build system.
-         By default the latest available DALI version will be downloaded.
-         Ignored, when TRITON_DALI_SKIP_DOWNLOAD is ON."
-)
+    "DALI version that should be downloaded by the build system.
+    By default the latest available DALI version will be downloaded.
+    Ignored, when TRITON_DALI_SKIP_DOWNLOAD is ON.")
+
 set(DALI_EXTRA_INDEX_URL "https://developer.download.nvidia.com/compute/redist" CACHE STRING
-        "URL of the Index, from which DALI will be downloaded by the build system.
-         Ignored, when TRITON_DALI_SKIP_DOWNLOAD is ON."
-)
+    "URL of the Index, from which DALI will be downloaded by the build system.
+    Ignored, when TRITON_DALI_SKIP_DOWNLOAD is ON.")
+
 set(DALI_DOWNLOAD_EXTRA_OPTIONS "" CACHE STRING
-        "Extra options for `pip download` call, that downloads DALI wheel.
-         Ignored, when TRITON_DALI_SKIP_DOWNLOAD is ON."
-)
+    "Extra options for `pip download` call, that downloads DALI wheel.
```

Comment:
Maybe you can print `DALI_DOWNLOAD_EXTRA_OPTIONS` as well?

Reply:
done